recipe/meta.yaml

```diff
@@ -60,12 +51,15 @@ outputs:
         - fonttools >=4.22.0
         - freetype >=2.3
         - kiwisolver >=1.0.1
+        - numpy >=1.20
         - {{ pin_compatible('numpy') }}
         - packaging >=20.0
         - pillow >=6.2.0
-        - pyparsing >=2.2.1
+        - pyparsing >=2.3.1
         - python-dateutil >=2.7
         - freetype
+        - tk >=8.6.11,<8.7  # [linux]
+        - importlib_resources >=5.2.0  # [py<310]
```

Comment:
Have you verified that nonw of the mentioned dependencies in run, which are not mentioned in host section, do not specify themself a run_export?  As if so, we need to add them to host section too, as otherwise pinning would be done incorrectly

Reply:
I have verified that none of the dependencies unmentioned in the host section specify themselves with run_export.

Reply:
thanks